### PR TITLE
Display user email alongside comment.

### PR DESCRIPTION
**Pull request by andres-holguin**
image
Made changes to:
- Store user email alongside their comment
- Create a Comment object so DataServlet returns ArrayList<Comment>
- Extract data from Comment object to display on the front end

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -43,6 +43,7 @@ function loadHTML(src, selector, position) {
     .then(response => response.text())
     .then(text => {
       document.querySelector(selector)?.insertAdjacentHTML(position, text);
+      console.log(`Loaded ${src} ${position}ing of ${selector}`);
```
Comment (nickKelman) [inline portfolio/src/main/webapp/script.js:46]: I assume you don't need this.

Comment (andres-holguin) [reply portfolio/src/main/webapp/script.js:46]: Correct, I was using this for debugging.
Funnily enough, before finding out I needed to disable caching, a print elsewhere actually made the loading more consistent. This makes me think it's a concurrency issue.
Should I remove this line?

Comment (nickKelman) [reply portfolio/src/main/webapp/script.js:46]: Yeah typically you try to keep debugging logic out of your master branch

Comment (andres-holguin) [reply portfolio/src/main/webapp/script.js:46]: Gotcha, I'll keep that in mind. Thanks :)

**Pull request merged**